src/ucp/core/ucp_worker.c

```diff
@@ -2500,7 +2500,11 @@ void ucp_worker_destroy(ucp_worker_h worker)
                  worker->flush_ops_count, worker);
     }
 
-    ucs_assert(worker->num_all_eps == 0);
+    if (worker->num_all_eps != 0) {
+        ucs_warn("not all endpoints (%u) were destroyed on worker %p that is"
+                 " being destroyed", worker->num_all_eps, worker);
```

Comment:
"worker %p: %u endpoints were not destroyed"

Reply:
done